### PR TITLE
Add message about setting discrete channel release flag

### PR DESCRIPTION
**Pull request by zentron**

### CODE REVIEW

**Comment by zentron** [comment]
[Deployed version highlighting not correct for channels on project dashboard](https://trello.com/c/na0B4EQ0/33-deployed-version-highlighting-not-correct-for-channels-on-project-dashboard)

File: docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting_discretechannels_[3.12.1,).partial.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Retention policies with Channels
+{#RetentionpolicyTentaclecleanupandtroubleshooting-Retentionpolicywithchannels}
+[Channels](/docs/key-concepts/projects/channels.md) can be used in Octopus to handle many different deployment scenarioes. In some cases you may have a hotfix channel in which deployments, as they are promoted through their environments, should be considered as overriding deployments from the default channel for the given enviroment. Alternatively you may be using channels to deploy feature branches which may involve having several concurrent releases active at any one time across different channels for the same environment. When using the feature branch type scenario, you will likely want retention policies to factor in the channel when deciding which previous deployment files should be removed from your Tentacle. This behaviour can be enabled for each project via the `Discrete Channel Release` flag at under `Deployment Target settings` on the {{Project,Process}} page.
```
Comment (matt-richardson) [inline docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting_discretechannels_[3.12.1,).partial.md]: * typo in 'scenarioes'.
* Needs a bit of re-wording - hard to grok it.
* wording for `Discrete Channel Release` is different to screenshot

File: docs/patterns/branching_discretechannels_[3.12.1,).partial.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+To prevent [retention policies](/docs/administration/retention-policies/index.md) for one channel from impacting deployments for another channel, you should set the `Discrete Channel Release` flag at under `Deployment Target settings` on the {{Project,Process}} page. Enabling this feature will also ensure that your project overview dashboard correctly shows which releases are current for each environment _in each channel_. Without this set, the default behaviour is for releases across channels to supersceed each other, for example in a hotfix scenario where the `3.2.2-bugfix` is expected to override the `3.2.2` release which may then be considered for retention policy cleanup.
```
Comment (matt-richardson) [inline docs/patterns/branching_discretechannels_[3.12.1,).partial.md]: screenshot says 'discrete channel releases' (missing trailing 's' in the words.
typo in 'supersceed' - 'supersede'

`Without this set, the default behaviour is for releases across channels to supersceed each other, for example in a hotfix scenario where the `3.2.2-bugfix` is expected to override the `3.2.2` release which may then be considered for retention policy cleanup. ` => `Without this set, the default behaviour is for releases across channels to supersede each other (for example, in a hotfix scenario where the `3.2.2-bugfix` is expected to override the `3.2.2` release, allowing `3.2.2` to be considered for retention policy cleanup).`

**Comment by johnsimons** [comment]
@zentron the filename `troubleshooting_discretechannels_[3.12.1,).partial.md` will not work given docs only publishes minor versions, so that will only be included in 3.13, is that ok ?

**Comment by zentron** [comment]
sounds like ill just change the partials to `3.12.0` merge then include comment that its just for `3.12.2+`

**Pull request merged**